library/erlang

```diff
@@ -0,0 +1,10 @@
+# maintainer: denc716@gmail.com (@c0b)
+
+18.1.3: git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 18
+18.1:   git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 18
+18:     git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 18
+latest: git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 18
+
+17.5.6.4: git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 17
+17.5:     git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 17
+17:       git://github.com/c0b/docker-erlang-otp@7a86ea4d35b99d1c40a9f2fe1470615e096c952a 17
```

Comment:
Did you want to bump this one more time before we merge to include some of these other changes? https://github.com/c0b/docker-erlang-otp/compare/7a86ea4d35b99d1c40a9f2fe1470615e096c952a...master